modules/installation-about-custom-aws-vpcs.adoc

```diff
@@ -0,0 +1,63 @@
+
+If you us a cluster with public access, you must create a public and a private subnet for each availability zone that your cluster uses. You must reserve these subnets for use by the {product-title} cluster. The installation program modifies your subnets to add the `kubernetes.io/cluster/.*: shared` tag, so your subnets must have at least one free tag slot available for it. Review the current link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-restrictions[Tag Restrictions] in the AWS documentation to ensure that the installation program can add a tag to each subnet that you specify.
+
+To ensure that the subnets that you provide to the installation program are suitable, it confirms the following data:
+
+* All the subnets that you specify exist.
+* You provide private subnets.
+* The subnet CIDRs belong to the machine CIDR that you specified.
+* Unless you use a private cluster, you must provide exactly one public and private subnet for each availability zone. Each availability zone contains no more than one public and one private subnet.
+* You provide a public subnet for each private subnet availability zone. Machines will not be provisioned in availability zones that you do not provide private subnets for.
+* If you are not using a private cluster, you provided a public subnet for each availability zone.
+
+If you destroy a cluster that uses an existing VPC, the VPC is not deleted. When you remove the{product-title} cluster from a VPC, the `kubernetes.io/cluster/.*: shared` tag is removed from the subnets that it used.
+
+[id="installation-about-custom-aws-permissions_{context}"]
+== Division of permissions
+
+Starting with {product-title} 4.3, you do not need all of the permissions that are required for an installation program-provisioned infrastructure cluster to deploy a cluster. This change mimics the division of permissions that you might have at your company: some individuals can create different resource in your clouds than others. For example, you might be able to create application-specific items, like instances, buckets, and load balancers, but not networking-related components such as VPCs, subnet, or ingress rules.
+
+The AWS credentials that you use when you create your cluster do not need the networking permissions that are required to make VPCs and networking core components within the VPC, such as subnets, routing tables, internet gateways, NAT, and VPN. You still need permission to make the application resources that the machines within the cluster require, such as ELBs, security groups, S3 buckets, and nodes.
+
+[id="installation-about-custom-aws-vpcs-isolation_{context}"]
+== Isolation between clusters
+
+If you deploy {product-title} to an existing network, the isolation of cluster services is reduced in the following ways:
+
+* You can install multiple {product-title} clusters in the same VPC.
+* ICMP ingress is allowed to entire network.
```

Comment:
`to entire network` -> `from the entire network`, because we're talking about ingress?